Tests/ComposableArchitectureTests/Support/ProcessInfo.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+extension ProcessInfo {
+  var isCI: Bool {
+    print("!!!")
+    dump(self.environment)
+    print("!!!")
```

Comment:
we wanna remove these before merging right?

```suggestion
```

Reply:
Still debugging.